v4/newrelic/sql_driver.go

```diff
@@ -187,10 +191,11 @@ func (w *wrapConn) Query(query string, args []driver.Value) (driver.Rows, error)
 
 // QueryContext implements QueryerContext.
 func (w *wrapConn) QueryContext(ctx context.Context, query string, args []driver.NamedValue) (driver.Rows, error) {
-	segment := w.bld.useQuery(query).startSegment(ctx)
+	startTime := time.Now()
 	rows, err := w.original.(driver.QueryerContext).QueryContext(ctx, query, args)
 	if err != driver.ErrSkip {
-		segment.End()
+		seg := w.bld.useQuery(query).startSegmentAt(ctx, startTime)
+		seg.End()
```

Comment:
For some reason, this update to the sql driver file wasn't included! I caught this through running some of the tests! In order to add this back in, I also needed to include a `txn.startSegmentAt` method.